src/fu-daemon.c

```diff
@@ -1990,33 +1990,43 @@ fu_daemon_daemon_method_call(GDBusConnection *connection,
 		return;
 	}
 	if (g_strcmp0(method_name, "GetBiosAttrs") == 0) {
-		g_autoptr(FuMainAuthHelper) helper = NULL;
+		g_autoptr(FuBiosAttrs) attrs = NULL;
+		gboolean authenticate = FALSE;
+
+		g_variant_get(parameters, "(b)", &authenticate);
```

Comment:
This doesn't feel right; I kinda thought this kind of thing could be done with FwupdFeatureFlags; maybe using `FWUPD_FEATURE_FLAG_REQUESTS` -- or maybe adding a `FWUPD_FEATURE_FLAG_NO_AUTH` or `FWUPD_FEATURE_FLAG_ANON` type thing.

Reply:
added new flag